CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Changed
 - Create generic DocRequest to better categorize ActionRequests ([#18269](https://github.com/opensearch-project/OpenSearch/pull/18269)))
+- Switch ingest-geoip and lang-painless modules to use the gradle version catalog ([#18316](https://github.com/opensearch-project/OpenSearch/pull/18316)))
```

Comment:
Build related refactorings like this that do not change functionality for users or plugins are probably better left out of the changelog.